lib/pliny/middleware/versioning.rb

```diff
@@ -44,13 +44,19 @@ def detect_api_version(env)
       env['HTTP_ACCEPT'] = media_types.join(', ')
 
       version ||= @default
-      set_api_version(env, version)
+      set_api_version(env, version, variant)
+      set_log_context(version, variant)
       @app.call(env)
     end
 
-    def set_api_version(env, version)
+    def set_api_version(env, version, variant)
       # API modules will look for the version in env
-      env["HTTP_X_API_VERSION"] = version
+      env["api.version"] = version
+      env["api.variant"] = variant
+    end
+
+    def set_log_context(version, variant)
+      Pliny.default_context[:api_version] = [version, variant].join(".")
```

Comment:
This should probably be merged into `Pliny::RequestStore.store[:log_context].merge!`. `Pliny.default_context` is for application specific things like app name and deployment.